pkg/core/native/policy.go

```diff
@@ -42,6 +43,18 @@ var (
 // Policy represents Policy native contract.
 type Policy struct {
 	interop.ContractMD
+	lock  sync.RWMutex
+	cache cache
+}
+
+type cache struct {
```

Comment:
`cache` doesn't look like a good package-wide name for a type that is only being used by `Policy`. And I would probably not create this struct at all, you don't pass it around in any way, so you don't need it, everything might stored in `Policy`.